provider/testdata/src/blob.rs

```diff
@@ -4,11 +4,17 @@
 
 use icu_provider_blob::StaticDataProvider;
 
+#[cfg(feature = "smaller_static")]
 const STATIC_STR_DATA: &[u8] = include_bytes!(concat!(
     env!("CARGO_MANIFEST_DIR"),
-    "/data/testdata.postcard"
+    "/data/decimal-bn-en.postcard"
 ));
 
+#[cfg(not(feature = "smaller_static"))]
+const STATIC_STR_DATA: &[u8] = include_bytes!(concat!(
+    env!("CARGO_MANIFEST_DIR"),
+    "/data/testdata.postcard"
+));
```

Comment:
Issue: I'm concerned that doing it this way could cause problems where any dependency that enables the `"smaller_static"` feature would replace the full data with the smaller data.

Suggestion: Add two functions or modules, one that returns the full data and one that returns the small data.

I don't think they need to be two separate features in `icu_testdata`.  I'd like if the feature could be in `capi` only, and that's where the choice between the two bundles is chosen.

Reply:
sounds good